cmd/local.go

```diff
@@ -266,8 +281,14 @@ var localCmd = &cobra.Command{
 func init() {
 	serverLogsCmd.Flags().BoolVarP(&follow, "follow", "f", false, "follow logs output")
 	localCmd.AddCommand(serverLogsCmd)
+
 	serverUpCmd.Flags().BoolVarP(&loginParam, "login", "l", false, "login to the local instance after starting it")
+	serverUpCmd.Flags().StringVarP(&config.DefaultConfig.Project, "create-project", "p", config.DefaultConfig.Project,
+		"create project after start")
+	serverUpCmd.Flags().StringVarP(&config.DefaultConfig.Branch, "create-branch", "b", config.DefaultConfig.Branch,
```

Comment:
Nit: same comment applies to --b / --branch I would only spell the behavior out in the help and not in the parameter. Not a blocker though, otherwise LGTM.